.github/workflows/publish-release.yml

```diff
@@ -89,34 +89,32 @@ jobs:
         env: 
           SETUPTOOLS_SCM_PRETEND_VERSION: ${{ steps.set_version.outputs.gitlint_version }}
 
-      - name: Publish gitlint-core (pypi.org)
-        run: hatch publish
-        working-directory: ./gitlint-core
-        env:
-          HATCH_INDEX_USER: ${{ secrets.PYPI_GITLINT_CORE_USERNAME }}
-          HATCH_INDEX_AUTH: ${{ secrets.PYPI_GITLINT_CORE_PASSWORD }}
+      - name: Publish gitlint-core 🐍📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages-dir: gitlint-core/dist/
+          password: ${{ secrets.PYPI_GITLINT_CORE_PASSWORD }}
         if: inputs.pypi_target == 'pypi.org'
 
-      - name: Publish gitlint (pypi.org)
-        run: hatch publish
-        env:
-          HATCH_INDEX_USER: ${{ secrets.PYPI_GITLINT_USERNAME }}
-          HATCH_INDEX_AUTH: ${{ secrets.PYPI_GITLINT_PASSWORD }}
+      - name: Publish gitlint 🐍📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages-dir: gitlint-core/dist/
```

Comment:
```suggestion
```
I think this should be the top-level `dist/` package, which [according to the docs](https://github.com/pypa/gh-action-pypi-publish#customizing-target-package-dists-directory) is the default value for `packages-dir`.